lib/ansible/modules/storage/netapp/na_ontap_volume.py

```diff
@@ -222,6 +222,17 @@
     - zh_tw.big5        Traditional Chinese Big 5
     - To use UTF-8 as the NFS character set, append '.UTF-8' to the language code
     version_added: '2.8'
+
+  qos_policy_group:
+    description:
+    - Allows a QoS policy group to be set on volume creation.
+    version_added: '2.9'
+
+  qos_adaptive_policy_group:
```

Comment:
same as the comments above, mainly the examples one that we would like added if possible :)

Reply:
updated to 'Specifies'